### DISCUSSION TITLE
How to make a game like this

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [axmolengine/axmol#2099] [General]
Hi,

How to make a game like this.

![image](https://github.com/user-attachments/assets/cc51d6c8-1c21-41df-b90d-a70fbd1d303b)

<img width="1239" alt="image" src="https://github.com/user-attachments/assets/864e4fe7-c67f-4926-b45e-c178b5e7b049">

<img width="1239" alt="image" src="https://github.com/user-attachments/assets/d73bf8e1-8608-424f-89e3-9cac375f43d5">

It get 30% of screen and other parts is transparent.

It can be a fullscreen game with PNG transparent to the rest?

Link: https://store.steampowered.com/app/3064030/Whimside/

Thanks.

**Comment by Arvant** [comment]
if you can work at the free space of screen show that it is not a transparent png. i think this is multi windows that customized by developer to not show title bars and also use multi windows like mdi forms in .net programms.

**Comment by DelinWorks** [comment]
your answer is borderless windows which glfw gladly supports, you can also render transparent things but that requires fiddling with glfw
